### ISSUE TITLE
vmctl: remote-read adds too many second units to throughput report

### ISSUE THREAD

**Issue by uhthomas** [VictoriaMetrics/VictoriaMetrics#4555]
### Describe the bug

It should say "samples/s" but instead vmctl wants to be a snake:

```
VM worker 0:↙ 0 samples/ssssss
VM worker 1:↙ 0 samples/ssssss
VM worker 2:↙ 9 samples/ssssss
VM worker 3:↙ 0 samples/ssssss
VM worker 4:↙ 343 samples/ssss
VM worker 5:↙ 11 samples/sssss
```

### To Reproduce

```
❯ go run github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl@latest remote-read --remote-read-src-addr=http://127.0.0.1:8080/prometheus --remote-read-step-interval=hour --remote-read-use-stream=true --vm-addr=http://127.0.0.1:8480 --vm-concurrency=6 --vm-account-id=0 --remote-read-filter-time-start=2022-08-18T00:00:00Z
```

### Version

v1.91.3

**Comment by dmitryk-dk** [comment]
Hi @uhthomas ! I tested your issue but can't reproduce it. Here is an example from my local machine.

```
 go run github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl@latest remote-read --remote-read-src-addr=http://127.0.0.1:9090 --remote-read-step-interval=hour --remote-read-use-stream=true --vm-addr=http://127.0.0.1:8428 --remote-read-filter-time-start=2022-08-18T00:00:00Z --vm-concurrency=6
Selected time range "2022-08-18 00:00:00 +0000 UTC" - "2023-07-10 09:53:52.56405 +0000 UTC" will be split into 7834 ranges according to "hour" step. Continue? [Y/n] 
VM worker 0:→ 127573 samples/s
VM worker 1:→ 143159 samples/s
VM worker 2:→ 127497 samples/s
VM worker 3:→ 128005 samples/s
VM worker 4:→ 127955 samples/s
VM worker 5:→ 140824 samples/s
Processing ranges: 7834 / 7834 [████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████] 100.00%
2023/07/10 11:54:06 Import finished!
2023/07/10 11:54:06 VictoriaMetrics importer stats:
  idle duration: 25.915730166s;
  time spent while importing: 13.564682875s;
  total samples: 8400390;
  samples/s: 619283.92;
  total bytes: 559.8 MB;
  bytes/s: 41.3 MB;
  import requests: 44;
  import requests retries: 0;
2023/07/10 11:54:06 Total time: 13.567530875s

```
<img width="1600" alt="Screenshot 2023-07-10 at 11 55 07" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/12189570/e8878ae3-9edf-4a7c-8707-bd1944ab4b9a">

Could you share more information about how I can reproduce it?

**Comment by uhthomas** [comment]
I think this happens when the number of samples/a decreases by a significant place (i.e from 100 to 99).

**Comment by dmitryk-dk** [comment]
Hi @uhthomas ! Found a bug, and created a separate issue ticket in the library which we are using to show progress.

**Comment by valyala** [comment]
The issue with superfluous `s` chars at the throughput report should be fixed in [vmctl v1.92.0](https://docs.victoriametrics.com/CHANGELOG.html). Closing the issue as fixed then.
